docs/content/en/interaction-manager/spatial-navigation.md

```diff
@@ -124,6 +124,16 @@ Switches to another area and focuses on the first element.
 
 Unfocuses the currently focused element in a navigatable area.
 
+### changeKeys()
+
+```{.javascript}
+spatialNavigation.changeKeys({ up: 'W', down: 's', left: 'a', right: 'd' }, { changeDefault: true });
+```
+
+Sets new keys for spatial navigation.  
+
+Accepts an optional options object as the second argument with one option: `changeDefault`. Defaults to `false`. If `true`, it overrides the default navigation keys with the provided ones.
```

Comment:
```suggestion
The method accepts an optional options object as a last argument. The available options are: 
* `changeDefault` - Boolean. Defaults to `false`. If `true`, it overrides the default navigation keys with the provided ones. If `false` the default keys won't be overridden and the provided ones will be used as well.
```

Or something similar to this.

Reply:
Doc has been altered in order to fit the logic for the newly decided flag `changeCurrentActiveKeys`